src/pages/Profile/editProfile/index.jsx

```diff
@@ -1,29 +1,28 @@
 import React, { useState } from 'react';
 import { handleChangeState } from '../../../functions';
-import { getUser, useEditUser } from '../../../Api';
+import { useEditUser } from '../../../Api';
 import { enqueueSnackbar } from 'notistack';
 
 export const EditProfile = () => {
 
-  const user = getUser();
-  const { editUser } = useEditUser();
+  const { axiosEditUser } = useEditUser();
+  
   const [editUserField, setEditUserField] = useState({
-    username: user.username,
+    username: '',
     password: '',
   });
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async(e) => {
     e.preventDefault();
     if (!editUserField.username || !editUserField.password) {
      return enqueueSnackbar('Must Fill Every Field.', { variant: 'warning' });
-    } else if (user.username === editUserField.username && user.password === editUserField.password) {
-     return enqueueSnackbar('Nothing Change.', { variant: 'warning' });
```

Comment:
You can extract the logic to check if the password and use name are the same into a service/separate file